modules/internal/String.chpl

```diff
@@ -575,13 +578,14 @@ module String {
   }
 
   pragma "no doc"
-  proc chpl_createStringWithLiteral(x: c_string, length:int) {
+  proc chpl_createStringWithLiteral(x: c_string, length:int, numCodepoints: int) {
```

Comment:
It's not part of your work, but while touching this part of the code you can also add a space between `length:` and `int`. That seems to be the "convention" for these functions.